tools/contrib/subset_data.py

```diff
@@ -0,0 +1,1096 @@
+        sl = StreamToLogger(sys.stdout, stdout_logger, logging.INFO, also_log_to_stream)
+        sys.stdout = sl
+
+    @classmethod
+    def setup_stderr(cls, also_log_to_stream=True):
+        """
+        Setup logger for stdout
+        """
+        stderr_logger = logging.getLogger('STDERR')
+        sl = StreamToLogger(sys.stderr, stderr_logger, logging.ERROR, also_log_to_stream)
+        sys.stderr = sl
+
+    def write(self, buf):
+        temp_linebuf = self.linebuf + buf 
+        self.linebuf = ''
+        for line in temp_linebuf.splitlines(True):
+            if line[-1] == '\n':
+                self.logger.log(self.log_level, line.rstrip())
+            else:
+                self.linebuf += line
+
+    def flush(self):
+        if self.linebuf != '': 
+            self.logger.log(self.log_level, self.linebuf.rstrip()) 
+        self.linebuf = ''
+
+
+
+
+def main ():
+
+    args = get_parser().parse_args()
+
+    # --------------------------------- #
+
+    today = date.today()
+    today_string = today.strftime("%Y%m%d")
+
+    pwd = os.getcwd()
+
+    log_file = os.path.join(pwd, today_string+'.log')
+
+    log_level =  logging.DEBUG
+    setup_logging(log_file, log_level)
+    log = logging.getLogger(__name__)
+
+    print("User = "+myname)
+    print("Current directory = "+pwd)
+
+    # --------------------------------- #
+
+    if (args.run_type == "point"):
+        print( "----------------------------------------------------------------------------")
+        print( "This script extracts a single point from the global CTSM inputdata datasets." )
+
+        #--  Specify point to extract
+        plon = args.plon
+        plat = args.plat
+
+        #--  Create regional CLM domain file
+        create_domain   = args.create_domain
+        #--  Create CLM surface data file
+        create_surfdata = args.create_surfdata
+        #--  Create CLM surface data file
+        create_landuse  = args.create_landuse
+        #--  Create single point DATM atmospheric forcing data
+        create_datm     = args.create_datm
+        datm_syr = args.datm_syr
+        datm_eyr = args.datm_eyr
+
+        crop_flag = args.crop_flag
+
+        site_name = args.site_name
+
+        #--  Modify landunit structure
+        overwrite_single_pft = args.overwrite_single_pft
+        dominant_pft         = args.dom_pft
+        zero_nonveg_landunits= args.zero_nonveg
+        uniform_snowpack     = args.uni_snow
+        no_saturation_excess = args.no_saturation_excess
+
+
+        #--  Create SinglePoint Object
+        single_point = SinglePointCase(plat, plon,site_name, 
+                            create_domain, create_surfdata, create_landuse, create_datm,
+                            overwrite_single_pft, dominant_pft, zero_nonveg_landunits, uniform_snowpack,
+                            no_saturation_excess)
+        single_point.create_tag()
+
+
+        print (single_point)
+
+        if crop_flag:
```

Comment:
@negin513 is it simple enough to set this flag for the two neon sites that need in in your NEON wrapper script? Maybe we just add a `crop` column to the .csv file with site information you're reading in?

Reply:
see also #1364

Reply:
@danicalombardozzi suggested interpreting this `if PFT_1 > 16:` throw crop flag